react/ProductName.tsx

```diff
@@ -76,6 +84,9 @@ function ProductName({
   skuName,
   brandName,
   productReference,
+  displayMode,
```

Comment:
Add a default value for this prop right here

Reply:
Done!